pt/lessons/specifics/plug.md

```diff
@@ -72,40 +78,49 @@ Ela recebe um `%Plug.Conn{}` struct como seu primeiro argumento, e é esperado q
 
 ## Configurando o módulo do projeto
 
-Dado que estamos criando uma aplicação Plug do zero, precisamos definir o módulo da aplicação.
-Atualize `lib/example.ex` para iniciar e superviosionar o Cowboy.
+Nós precisamos dizer para a nossa aplicação iniciar e supervisionar o servidor web Cowboy when o app estiver de pé.
+
+Nós vamos fazer isso com a função [`Plug.Cowboy.child_spec/1`](https://hexdocs.pm/plug_cowboy/Plug.Cowboy.html#child_spec/1).
+
+Essa função espera três opções:
+
+* `:scheme` - HTTP ou HTTPS como um atom (`:http`, `:https`)
+* `:plug` - O módulo plug que deve ser usado como a interface para o servidor web. Você pode especificar o nome do módulo, como `MyPlug`, ou uma tupla com o nome do módulo e opções `{MyPlug, plug_opts}`, onde `plug_opts` é passada para a função `init/1` do seu módulos plug.
```

Comment:
At the of the phrase, I believe there is a small problem with the plural of "módulos".